### PR TITLE
Fixed some API urls that have changed, and incorrect createUser 201 response behaviour

### PR DESCRIPTION
**Pull request by ian-sorbello-transmit**
Working off the EU cloud, I found that the auth-hub sample had some incorrect pointers. 

1. Some APIs pointed to the US API cloud even when the Base URL was configured to EU
2. The createUser function in sign up expected a 200 response, otherwise views the sign up response as an error. CreateUser API now returns 201 as a successful create, so I've updated the code to expect 201
3. There were some missing /cis/ parts of the URL during callbacks on creating and logging in.

Overall, very minimal changes. There's a chance these updates may also be in another branch as part of general maintenance, so possibly this PR may not be needed.

### CODE REVIEW

File: authentication-hub/backend/lib/management.js [hunk 1]
```diff
@@ -22,7 +22,8 @@ export async function getUser(userId) {
  * See: https://developer.transmitsecurity.com/guides/user/manage_user_sessions/#step-5-logout-session
  */
 export async function logout(accessToken) {
-  const url = `https://api.userid.security/v1/auth/logout`; /* <-legacy. To be replaced with: common.config.apis.logout */
+  const url =
+    common.config.apis.logout; /* <-legacy. To be replaced with: common.config.apis.logout */
```
Comment (ts-ron) [inline authentication-hub/backend/lib/management.js:26]: Can remove the comment now :)

File: staging.env [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# duplicate and rename this file to .env
```
Comment (ts-ron) [inline staging.env:1]: Need to remove this file right? contains config and secrets

**Review by ts-ron — CHANGES_REQUESTED** [review]
LGTM - thanks :)
Need to git remove the config file (and add to .gitignore or something)

File: staging.env (hunk 2 above)
Comment (ian-sorbello-transmit) [reply staging.env:1]: ooft. I explicitly didn't add the file into the commit, but it made its way in somehow. Yep, removing.

**Comment by ian-sorbello-transmit** [comment]
@ts-ron - new update pushed. thanks for review. ps: this was my first git PR. It's fiddly working this stuff out for the first time..

**Review by ts-ron — APPROVED** [review]
LGTM 🥳

**Pull request merged**